djangosaml2idp/views.py

```diff
@@ -114,25 +116,35 @@ def get_authn(self, req_info=None):
         return broker.get_authn_by_accr(req_authn_context)
 
     def build_authn_response(self, user, authn, resp_args):
-        name_id_formats = [resp_args.get('name_id_policy').format] or self.IDP.config.getattr("name_id_format", "idp") or [NAMEID_FORMAT_UNSPECIFIED]
+        name_id_formats = [resp_args.get('name_id_policy').format] or \
+                           self.IDP.config.getattr("name_id_format", "idp") or \
+                           [NAMEID_FORMAT_UNSPECIFIED]
         authn_resp = self.IDP.create_authn_response(
             authn=authn,
             identity=self.processor.create_identity(user, self.sp['config']),
             userid=self.processor.get_user_id(user, self.sp['config']),
-            name_id=NameID(format=name_id_formats[0], sp_name_qualifier=self.sp['id'], text=self.processor.get_user_id(user, self.sp['config'])),
-            sign_response=self.sp['config'].get("sign_response") or self.IDP.config.getattr("sign_response", "idp") or False,
-            sign_assertion=self.sp['config'].get("sign_assertion") or self.IDP.config.getattr("sign_assertion", "idp") or False,
+            name_id=NameID(format=name_id_formats[0],
+                           sp_name_qualifier=self.sp['id'],
+                           text=self.processor.get_user_id(user,
+                                                           self.sp['config'])),
+            sign_response=self.sp['config'].get("sign_response") or
+                          self.IDP.config.getattr("sign_response", "idp") or False,
+            sign_assertion=self.sp['config'].get("sign_assertion") or
+                           self.IDP.config.getattr("sign_assertion", "idp") or False,
             **resp_args)
         return authn_resp
 
     def create_html_response(self, request, binding, authn_resp, destination, relay_state):
+        """ Login form for SSO
```

Comment:
Adding the short docstring here is a good change, I'm not sure about the other code style changes though. I put more info in a comment on the main PR thread.

Reply:
I done these. Regarding coding style I think that a pep8 compliance is better then personal tastes, bytheway It would be urgent to write this in CONTRIBUTING.md